src/components/picker/PickerItemsList.tsx

```diff
@@ -35,7 +35,7 @@ const PickerItemsList = (props: PickerItemsListProps) => {
     testID
   } = props;
   const context = useContext(PickerContext);
-  const [wheelPickerValue, setWheelPickerValue] = useState<PickerSingleValue>(context.value ?? items?.[0].value);
+  const [wheelPickerValue, setWheelPickerValue] = useState<PickerSingleValue>(context.value ?? items?.[0]?.value);
```

Comment:
Is this related to the main issue?
Can you provide an example?

Reply:
Yes it related.
When searching and there are no items matching the search it cause an issue for me and crash with error of undefined value.

Reply:
Go to the PickerScreen, in the first example change it from children to items:
```
  <Picker
            placeholder="Favorite Language"
            floatingPlaceholder
            value={this.state.language}
            enableModalBlur={false}
            onChange={item => this.setState({language: item})}
            topBarProps={{title: 'Languages'}}
            // style={{color: Colors.red20}}
            showSearch
            searchPlaceholder={'Search a language'}
            searchStyle={{color: Colors.blue30, placeholderTextColor: Colors.grey50}}
            // onSearchChange={value => console.warn('value', value)}
            items={longOptions}
          />
 ```
 Enter some `bla-bla` phrase in the search box and see the difference.